tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "include": ["next.d.ts", "**/*.ts", "**/*.tsx"],
+    "baseUrl": ".",
+    "compilerOptions": {
+      //"declaration": false,
```

Comment:
why?

Reply:
to fix the composite file error, need to declare a base url.